### PR TITLE
README update

### PR DESCRIPTION
**Pull request by TohidHeshmati**
not all programmers might have `jq` or `ngrok` installed.
I wrote instructions to install them. on Ubuntu and mac to run shell script more smoothly :)

### CODE REVIEW

**Comment by TohidHeshmati** [comment]
corrected some

**Comment by TohidHeshmati** [comment]
I seriously doubt this CI (my unpopular opinion ;) )
in first commit everything saw perfect to my human eye and after manipulating a lot I managed to pass its issues but now that I have passed its check list, the README looks awful.

File: README.md [hunk 1]
```diff
@@ -33,3 +33,19 @@ Below are some ideas I'll implement in the comming days, no promise to respect t
 -   [ ]  Slack chatbot connector
 -   [ ]  Design plugin to make the chatbot extensible
 -   [ ]  Some plugins out of the box
+
+------  
+to run the shell script called `set_telegram_webhook.sh` following steps might be needed:
```
Comment (s-soroosh) [inline README.md]: Add a title here like: 
How to run the example.

Comment (TohidHeshmati) [reply README.md]: thanks for handling it.

**Pull request merged**